assets/src/styles/blocks/Covers/TakeActionCovers.scss

```diff
@@ -268,13 +265,15 @@
 }
 
 .cover-card-btn {
+  --block-covers--card-button-- {
+    display: none;
```

Comment:
I added this to test out toggling the hover behavior with a variable. It works, but because we put other styles only on the `:hover` selector, it looks very different. We could get around this like described in this comment: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/523/files?diff=split&w=1#r612283529

Reply:
Ended up moving these rules already, after verifying that doesn't affect child themes.